content/v2.0/reference/api/client-libraries/python.md

```diff
@@ -56,7 +58,7 @@ Here, we simply pass the three variables we have already defined.
 
 ```python
 client = InfluxDBClient(
-    url="http://localhost:9999",
+    url="https://us-west-2-1.aws.cloud2.influxdata.com",
```

Comment:
This should actually stay `http://localhost:9999`, that way it will get replaced by the InfluxDB URL selection.